Hestia/Hestia/frontend/DevicesScreen/ActivatorScreen/TableSourceActivators.cs

```diff
@@ -54,10 +54,15 @@ public override UITableViewCell GetCell(UITableView tableView, NSIndexPath index
 
                 // Set the switch's state to that of the device.
                 slider.Value = (float)act.State.RawState;
-                slider.ValueChanged += delegate (object sender, EventArgs e)
+                slider.TouchUpInside += delegate (object sender, EventArgs e)
                 {
-                    act.State = new ActivatorState<object>(slider.Value, "float");
+                    //Console.WriteLine(slider.Value);
+                    act.State.RawState = slider.Value;
```

Comment:
This does not update the state on the server. The way it was done before is the correct way of doing it.